server/routes/api/documents/documents.test.ts

```diff
@@ -2955,6 +2955,35 @@ describe("#documents.delete", () => {
     expect(deletedDoc?.deletedAt).toBeTruthy();
   });
 
+  it("should delete a draft under deleted collection", async () => {
```

Comment:
```suggestion
  it("should not delete draft under deleted collection", async () => {
```

Reply:
This is for when attempt is made to delete the draft document, not collection.